.circleci/config.yml

```diff
@@ -9,17 +9,15 @@ jobs:
       tag: 'lts'
     steps:
       - checkout
-      - run: echo "registry=$SYM_NPM_READ_REPO" > ~/.npmrc
-      - run: echo "_auth=$SYM_NPM_TOKEN" >> ~/.npmrc
-      - run: echo "always-auth=true" >> ~/.npmrc
       - run: yarn
       - run: yarn lint
       - run: yarn check-circular
       - run:
           command: yarn test-once
           environment:
-              NODE_OPTIONS: --max_old_space_size=4096
+            NODE_OPTIONS: --max_old_space_size=4096
       - run: yarn build
+      - run: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
```

Comment:
Are we sure that `${NPM_TOKEN}` value does not appear in CircleCI job logs?

Reply:
more over do we need a NPM token to access the public registry?

Reply:
ah below we public

Reply:
@symphony-thibault CircleCI knows how to handle secrets :)

Reply:
@symphony-thibault  Yes, the token will not appear in the logs. the content of the .npmrc file is not displayed. Even if we try, then it will display with '*****' instead of the value. I'll already tried ;)  (For more info: https://circleci.com/docs/2.0/env-vars/#example-configuration-of-environment-variables)

It's seems to be the good way to configure the .npmrc with CircleCI: See https://circleci.com/blog/publishing-npm-packages-using-circleci-2-0/